app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Send me reset password instructions" %>
```

Comment:
- [ ] **[OPTIONAL]** Password reset works just fine - the link is sent and the password can be reset, however, I think it would be better if we don't get this page after requesting a reset.

  You can fix this by changing this line to this:
  ```
  <%= f.submit "Send me reset password instructions", data: { turbo: false } %>
  ```

  ![fjff](https://user-images.githubusercontent.com/36542986/207539487-72077c6e-0ab2-4e1d-824a-e8933b5f19ed.jpg)